FILE packages/website/src/routes/index.tsx
RANGE {"path":"packages/website/src/routes/index.tsx","old_lines":[223,227],"new_lines":[223,227],"diff_block_lines":[5,11],"hunk_header":"@@ -223,5 +223,5 @@ function GetStarted() {","substantive_changed_lines":1}
@@ -223,5 +223,5 @@ function GetStarted() {
         >
           <AppleIcon className="h-4 w-4" />
-          Request iOS
+          iOS
         </a>
         <a
